pkg/internal/discover/attacher_linux.go

```diff
@@ -29,19 +30,22 @@ func (ta *TraceAttacher) mountBpfPinPath() error {
 	return ta.bpfMount(ta.pinPath)
 }
 
-func (ta *TraceAttacher) unmountBpfPinPath() {
-	if err := unix.Unmount(ta.pinPath, unix.MNT_FORCE); err != nil {
-		ta.log.Warn("can't unmount pinned root. Try unmounting and removing it manually", "error", err)
-		return
+func UnmountBPFFS(pinPath string, log *slog.Logger) {
+	if err := unix.Unmount(pinPath, unix.MNT_FORCE); err != nil {
```

Comment:
what's the use-case behind `MNT_FORCE`? I wonder if we should be using `MNT_DETACH` instead...